### ISSUE TITLE
It's not working in browser

### ISSUE THREAD

**Issue by iagopiimenta** [lrsjng/modulejs#5]
Babel remapping ES6 "this" to undefined.

```js
// ...
(function (root, factory) {
    // istanbul ignore else
    if ((typeof exports === 'undefined' ? 'undefined' : _typeof(exports)) === 'object') {
        module.exports = factory();
    } else {
        root.modulejs = factory(); // throw: Uncaught TypeError: Cannot set property 'modulejs' of undefined
    }
})(undefined, function () { // <-------------------------
    // eslint-disable-line no-invalid-this
    var OBJ_PROTO = Object.prototype;
// ...
```

**Comment by lrsjng** [comment]
@iagopiimenta thanks for reporting! missed that completely.. added a quick fix in c5b52b71e7c423544d84d3824dd6a5b922524c1c would be create, if you could recheck it. Project page should update in about ~5min. Thank you!

**Comment by iagopiimenta** [comment]
Great, it's working now !! Thanks for the quick feedback.
